boards/nucleo-f091/include/periph_conf.h

```diff
@@ -156,6 +156,19 @@ extern "C" {
 #define GPIO_5_IRQ          EXTI4_15_IRQn
 /** @} */
 
+/**
+ * @name RTC configuration
+ * @{
+ */
+/**
+ * Uncomment the following line if your Nucleo has MB1136 C-02 or
+ * MB1136 C-03 -sticker on it. These newer versions have the required LSE
+ * oscillator provided on the X2 slot.
+ * See Nucleo User Manual UM1724 section 5.6.2.
+ */
+/* #define RTC_NUMOF           (1U) */
```

Comment:
Why is it commented out? I thought this oscillator **is** available on the nucleo-f091